test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj

```diff
@@ -11,6 +11,7 @@
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="4.19.4" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage" Version="1.6.0" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="10.0.1" />
```

Comment:
This reference makes me a little nervous since it's another flavor of the WindowsAzure.Storage nuget that we're already importing via `Microsoft.Azure.DurableTask.AzureStorage`. Not a big deal, but I wonder if we should avoid adding this dependency to avoid the possibility of conflicts.

Reply:
Good point. Just tried the tests without adding that dependency and they still pass, so I will remove this addition.